ballerina/Module.md

```diff
@@ -0,0 +1,42 @@
+## Overview
+
+[Discord](https://support.discord.com/hc/en-us/articles/360045138571-Beginner-s-Guide-to-Discord) is a popular communication platform designed for creating communities and facilitating real-time messaging, voice, and video interactions over the internet.
+
+The Ballerina Discord connector offers APIs to connect and interact with the [Discord REST API v10](https://discord.com/developers/docs/reference)
+
+## Setup guide
+
+Follow these steps to create a discord developer account
+
+### Step 1: Login to Discord Developer's Page
+
+<img src="https://github.com/HussainLatiff/module-ballerinax-discord/blob/main/docs/setup/resources/discord-dev-page.png?raw=true">
+
+Visit the [Discord website for developers](https://discord.com/login?redirect_to=%2Fdevelopers) by logging into your discord account. 
+
+If you do not have a discord account already, [**create** a new discord account](https://discord.com/login) by clicking on the `Register` hyperlink below the `Log In` button when opening the Discord Developer Page.
+
+<img src= "https://github.com/HussainLatiff/module-ballerinax-discord/blob/main/docs/setup/resources/create-acc.png?raw=true">
+
+Complete the account creation process by including the relavant information in the given fields.
+
+### Step 2: Make a New Discord Application
+
+<img src="https://github.com/HussainLatiff/module-ballerinax-discord/blob/main/docs/setup/resources/make-new-app.png?raw=true">
+
+Once in the discord developer portal is open, click on the `New Application` button as displayed above to start the process.
+
+### Step 3: Name the Discord Application 
+
+<img src="https://github.com/HussainLatiff/module-ballerinax-discord/blob/main/docs/setup/resources/create-app.png?raw=true">
+
+1. Proceed by giving the Discord Application a name and click on the terms of service.
+
+2. Finally complete the naming process by clicking on the `next` button.
+
+### Step 4: Obtain the Client ID and Client Secret
+
+<img src="https://github.com/HussainLatiff/module-ballerinax-discord/blob/main/docs/setup/resources/obtain-client-id.png?raw=true">
+
+Under the `OAuth2` section found on the left-sided list, locate the Client's Information as shown on the screen. To implement the functionalities provided by Discord's API, you will need the Client ID and Client Secret.
+
```

Comment:
Let's include the `Quickstart` section as well.

Reply:
Ack, Quick start information has been added